node/src/types/deploy.rs

```diff
@@ -170,6 +172,19 @@ pub enum DeployValidationFailure {
         got: usize,
     },
 
+    /// Missing payment amount.
+    #[error("missing payment amount")]
+    MissingPaymentAmount,
```

Comment:
```suggestion
    #[error("missing payment argument amount")]
    MissingPaymentAmountArgument,
```

I think it will make it clearer to the users.

Reply:
Applied suggestion